src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -568,33 +568,25 @@ private static IsolatedStorageFile GetStore(IsolatedStorageScope scope)
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, Type applicationEvidenceType)
         {
             // Scope MUST be Application
-
-            // https://github.com/dotnet/corefx/issues/10935
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CAS); // https://github.com/dotnet/corefx/issues/10935
         }
 
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, object applicationIdentity)
         {
             // Scope MUST be Application
-
-            // https://github.com/dotnet/corefx/issues/10935
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CAS); // https://github.com/dotnet/corefx/issues/10935
```

Comment:
Even with the PNSE, I suggest this first match desktop and throw an arg exception if applicationIdentity is null:
```C#
if (applicationIdentity == null)
{
    throw new ArgumentNullException(nameof(applicationIdentity));
}
```
but regardless, what is the applicationIdentity used for?  As with above, if it's purely to influence CAS, since we don't have CAS, could/should this just delegate to GetStore(scope) rather than throwing a PNSE?